preprocessors/togeojson-gpx.preprocessor.js

```diff
@@ -95,15 +95,29 @@ module.exports = function(infile, outdirectory, callback) {
     // Create metadata file for original gpx source
     var metadatafile = path.join(outdirectory, '/metadata.json');
     digest(infile, function(err, metadata) {
-      if (err) return callback(err);
       fs.writeFile(metadatafile, JSON.stringify(metadata), function(err) {
-        if (err) return callback(err);
-        return createIndices(callback);
+        if (err) throw err;
+        createIndices(function(err) {
+          if (err) throw err;
+          archiveOriginal(function(err) {
+            if (err) throw err;
+            return callback(err);
```

Comment:
Do we always want to return an error? Usually this would look like `return callback()` or `return callback(null, something)`.